react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+
+export function HotButton({ text }) {
+  const [index, setIndex] = useState(0);
+
+  function handleClick() {
+    setIndex(index + 1);
+  }
+  if (index < 3) {
```

Comment:
You should precompute the class name and pass it to the `className` property. In your code you repeat the JSX element over and over again. Extract the class name and you can have just one line of code for the JSX.